lib/Setup/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const Mosaic = require('@openstfoundation/mosaic.js');
+const brandedtoken = require('./brandedtoken');
+const utilitybrandedtoken = require('./utilitybrandedtoken');
+
+/**
+ * @file The Setup module provides an abstraction layer to simplify
+ * deployment and setup of a branded token.
+ */
+
+module.exports = {
+  organization: Mosaic.ContractInteract.Organization.setup,
```

Comment:
seems out of line with L14 and L15; organization?

Reply:
Yes, because organization setup is taken for `mosaic.js` contract interact and the remaining two are actually setup submodule from this repository. 

For the consumers, usage for all three is consistent.